docs/Design/fpp-json-dict.md

```diff
@@ -552,41 +537,35 @@ module MyComponents {
 ## Telemetry Channels
 | Field | Description | Options | Required |
 | ----- | ----------- | ------- | -------- |
-| `identifier` | **String** qualified name of the telemetry channel | Period separated **String** | true |
+| `name` | **String** qualified name of the telemetry channel | Period separated **String** | true |
 | `description` | **String** annotation of channel | **String** | true |
 | `type` | [Type Name](#type-names) the telemetry channel | [Type Name](#type-names) | true |
-| `numericIdentifier` | **Number** representing numeric identifier | **Number** | true |
-| `telemetryUpdate` | **String** representing when the telemetry channel can update | always, on change | false |
-| `formatString` | **String** format with a single argument (the telemetry channel) | **String** | false |
+| `id` | **Number** representing numeric identifier | **Number** | true |
+| `telemetryUpdate` | **String** representing when the telemetry channel can update | always, on change | true |
+| `format` | **String** format with a single argument (the telemetry channel) | **String** | false |
 | `limit` | **JSON dictionary** consisting of low and high limits | **JSON dictionary** | false |
 
 Example FPP model with JSON representation:
 ```
-module MyModule {
-    active component FirstComponent {
-        @ Telemetry channel 1
-        telemetry Channel1: F64 \
-            id 0x100 \
-            update on change \
-            low { yellow -1, orange -2, red -3 } \
-            high { yellow 1, orange 2, red 3 }
-        
-        ...
-    }
-}
+@ Telemetry channel 1
+telemetry Channel1: F64 \
+    id 0x100 \
+    update on change \
+    low { yellow -1, orange -2, red -3 } \
+    high { yellow 1, orange 2, red 3 }
 ```
 
 ```json
 [
     {
-        "identifier": "MyModule.FirstComponent.Channel1",
+        "name": "c1.Channel1",
         "description": "Telemetry channel 1",
         "type": {
             "type": "F64",
```

Comment:
Is this `type` sub-dictionary here out of date?

Reply:
Good catch, should be resolved now